scripts/run_all.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
+#  Raw-raw data is in /nfs/lsst2/photocalData/data/ctio0m9 - NEVER TOUCH THIS DATA!
+#  Data that has been *copied* and then had the headers *patched* by the pre-ingest script
+#  is currently in /nfs/lsst2/photocalData/data/ctio0m9_sanitised
+#  if you rerun the pre-ingest script it will make you another copy
+
+export LD_LIBRARY_PATH=$LSST_LIBRARY_PATH
+export DYLD_LIBRARY_PATH=$LSST_LIBRARY_PATH
+
+CTIO_DATA_DIR=/nfs/lsst2/photocalData/data/ctio0m9_sanitised
+REPO_DIR=$HOME/ctio_repo
+whoami=$(whoami)
+batchArgs="--batch-type none"
+
+#-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-
+
+rm -rf $REPO_DIR
+mkdir -p $REPO_DIR
+
+echo "lsst.obs.ctio0m9.Ctio0m9Mapper" > $REPO_DIR/_mapper
+
+# Ignore these entirely - I think these are all duplicates, or have the wrong size CCD
```

Comment:
Please check on these cases and delete this comment before committing to master.